vtStorAtaManaged/vtStorAtaManaged.vcxproj

```diff
@@ -50,6 +51,7 @@
   <PropertyGroup Label="Configuration" Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
     <PlatformToolset>v120</PlatformToolset>
     <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <UseDebugLibraries>false</UseDebugLibraries>
```

Comment:
We have some issues with Debug|x64, others mode is ok ? If we change the config, please check